site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2705,6 +2705,10 @@ public function deleteComponents(array $components) {
      * @param \app\models\gradeable\Gradeable $gradeable The gradeable to insert
      */
     public function createGradeable(\app\models\gradeable\Gradeable $gradeable) {
+        $regrade_date=null;
+        if($gradeable->getRegradeRequestDate()!==null){
+            $regrade_date=DateUtils::dateTimeToString($gradeable->getRegradeRequestDate());
+        }
```

Comment:
This can be done in-line with the array construction (see lines 2722 and 2723 for an example)